MBBSEmu/DOS/Interrupts/Int21h.cs

```diff
@@ -0,0 +1,280 @@
+	                         1 - Standard Output Device - can be redirected (STDOUT)
+	                         2 - Standard Error Device - can be redirected (STDERR)
+	                         3 - Standard Auxiliary Device (STDAUX)
+	                         4 - Standard Printer Device (STDPRN)
+                         */
+
+                        if (fileHandle <= 2)
+                            Console.WriteLine(Encoding.ASCII.GetString(dataToWrite));
+
+                        break;
+                    }
+                case 0x44:
+                    {
+                        /*
+                            INT 21 - AH = 44H DOS Get Device Information
+                            
+                            Sub-Function Definition is in AL
+                         */
+                        switch (_registers.AL)
+                        {
+
+                            case 0x0:
+                                {
+                                    /*
+                                        INT 21 - AX = 4400h DOS 2+ - IOCTL - GET DEVICE INFORMATION
+                                        BX = file or device handle
+                                        Return: CF set on error
+                                         AX = error code
+                                        CF clear if successful
+                                         DX = device info
+                                     */
+
+                                    //Device
+                                    if (_registers.BX <= 2)
+                                    {
+                                        _registers.DX = 0;
+
+                                        _registers.DX |= 1; //STD Input
+                                        _registers.DX |= 1 << 1; //STD Output
+                                        _registers.DX |= 1 << 4; //Reserved? DOSBox sets it
+                                        _registers.DX |= 1 << 6; //Not EOF
+                                        _registers.DX |= 1 << 7; //IS Device
+                                        _registers.DX |= 1 << 15; //Reserved? DOSBox sets it
+                                    }
+                                }
+                                break;
+                        }
+
+                        break;
+                    }
+                case 0x47:
+                    {
+                        /*
+                            DOS 2+ - GET CURRENT DIRECTORY
+                            DL = drive (0=default, 1=A, etc.)
+                            DS:DI points to 64-byte buffer area
+                            Return: CF set on error
+                            AX = error code
+                            Note: the returned path does not include the initial backslash
+                         */
+                        _memory.SetArray(_registers.DS, _registers.SI, Encoding.ASCII.GetBytes("BBSV6\\\0"));
```

Comment:
Add a TODO here to implement this properly?

Reply:
It's not wrong -- technically. All MBBS Modules ran within the context of the same directory as MajorBBS (`BBSV6` in most cases), so this wouldn't change as addon utilities also ran within the same folder.

Reply:
Should be OK for now, though it might impact actual file operations down the line. Not pressing at this time at least